ivy_tests/test_ivy/test_frontends/test_torch/test_linalg.py

```diff
@@ -368,3 +368,37 @@ def test_torch_svd(
         atol=1e-2,
         ground_truth_backend=frontend,
     )
+
+
+@handle_frontend_test(
+     fn_tree="torch.linalg.eigh",
+     dtype_and_x=_get_dtype_and_square_matrix(),
+ )
+def test_torch_eigh(
+     *,
+     dtype_and_x,
+     as_variable,
+     with_out,
+     num_positional_args,
+     native_array,
+     on_device,
+     fn_tree,
+     frontend,
+ ):
+     dtype, x = dtype_and_x
+     helpers.test_frontend_function(
+         input_dtypes=dtype,
+         as_variable_flags=as_variable,
+         with_out=with_out,
+         num_positional_args=num_positional_args,
+         native_array_flags=native_array,
+         frontend=frontend,
+         fn_tree=fn_tree,
+         on_device=on_device,
+         input=x,
+     )
+
+
```

Comment:
We should have only 1 new line after the end of function definition

Reply:
ret_np = array([0.25, 1.75]), ret_from_gt_np = array([0.5, 1.5])

This is what is causing the error. Could you provide me some idea as to why this is happening?

**I have made sure to have only 1 new line at the end of the function. Thank you.

Reply:
Hello @aymuos15, I'm not entirely sure, I'd suggest trying to debug your code and see where that breaks, the log basically means the return from the framework isn't the same as the return of our frontend function implementation.

Reply:
Hello @CatB1t, thank you for the suggestion. I will try working on this.